app/controllers/api/v3/public/channels_controller.rb

```diff
@@ -11,6 +11,39 @@ def totals
     render(json: statistical_totals_json, status: 200)
   end
 
+  # Takes an array of channel identifiers and retuns a dictionary of channel identifiers as keys and true/false as values
+  def allowed_countries
+    channels = params[:channel_ids].map do |id|
```

Comment:
I would use a single query here https://stackoverflow.com/a/39035889

Reply:
Created issue to resolve the `channel_identifier` problem: https://github.com/orgs/brave-intl/projects/13#card-86385813